scripts/clustering/clustering.py

```diff
@@ -23,7 +28,7 @@ def cluster(smile_keys, cutoff=0.15):
     for i in range(0, nfps):
         murcko = convert(smile_keys[i])
         mols = Chem.MolFromSmiles(murcko)
```

Comment:
Not on you but if you could rename this variable mol so it's less confusing that would be awesome